.baseline/idea/intellij-java-palantir-style.xml

```diff
@@ -0,0 +1,461 @@
+<project version="4">
```

Comment:
@iamdanfox would it be reasonable to allow the name of this file to be configurable? I was looking at the Baseline source code and the file name is [hardcoded to this path](https://github.com/palantir/gradle-baseline/blob/9c95a967943b375f7da719b3265d2796c4a8c72f/gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy#L65). But including a reference to `Palantir` seems strange for projects that want to use baseline that don't find their origins with Palantir itself.

What I'd like to do instead is to run `./gradlew baselineUpdateConfig` to get this template, but then be able to rename this file and reference the renamed path in `build.gradle`.